Gemfile.lock

```diff
@@ -371,4 +371,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.3.23
+   2.3.22
```

Comment:
Reverted to 2.3.22 to try and replicate the failure in CI, and what information we can get from there